data/tribes/workers/frisians/builder/init.lua

```diff
@@ -56,7 +56,7 @@ tribes:new_worker_type {
          hotspot = {9, 24},
          sound_effect = {
             path = "sound/hammering/hammering",
-            priority = 64
+            priority = 50
```

Comment:
is there a reason to have a differing syntax here. at least I would have expected this to read with a % as well

Reply:
I haven't found a good place where to stick the common algorithm yet. Let's do this in a follow-up branch.

Reply:
ok